script.js

```diff
@@ -0,0 +1,39 @@
+let bankOpen = true;
+let balance = 500;
+
+do {
+  const welcome = prompt("Select (Q)uit, (W)ithdraw, (D)eposit, or (B)alance");
+
+  switch(welcome) {
+    case "q":
+      bankOpen = false;
+      alert('Thank you for banking with us!');
+      break;
+    case "b":
+      text = 'Your balance is $' + balance.toString();
+      alert(text);
+      break;
+    case "d":
+      const depositAmount = Number(prompt('How much would you like to deposit?'));
+      balance = depositAmount + balance;
+      if (balance >= 50000) {
```

Comment:
this should come before you do the math on the deposit, otherwise you can still add money to it. You could put the balance = depositAmount + balance in that if statement